app/models/playlist.js

```diff
@@ -6,37 +8,60 @@ const Playlist = Backbone.Model.extend({
     defaults: {
         _id: undefined,
         title: '',
-        tracks: '',
+        tracks: new Tracks([], {}),
```

Comment:
This one won't work: you're creating a new `Tracks` instance as a default value, so this one will be used by all `Playlist` instances (it's an object so it's shared by reference across all instances). Create it in the `initialize` method instead to ensure each `Playlist` have its own `Tracks` instance.